py/torch_migraphx/fx/tracer/acc_tracer/acc_ops.py

```diff
@@ -261,6 +261,26 @@ def select_scatter(*, input, src, dim, index):
 def linear(*, input, weight, bias):
     return nn.functional.linear(input=input, weight=weight, bias=bias)
 
+@register_acc_op_mapping(
+    op_and_target=("call_function", torch.nn.functional.nll_loss),
+    arg_replacement_tuples=[
+        ("input", "input"),  
+        ("target", "target"),
+        ("weight",  "weight", this_arg_is_optional),
+        ("size_average", "size_average", this_arg_is_optional),
+        ("reduce", "reduce", this_arg_is_optional),
+        ("reduction", "reduction", this_arg_is_optional),
+        ("ignore_index" , "ignore_index" , this_arg_is_optional)
+    ],
+)
+# defines the list of arguments recognized when we define a forward 
+# function in a torch.nn.Module .
+@register_acc_op
+def nll_loss_forward(*, input, target, weight=None, reduce=None, reduction='mean', size_average=None, ignore_index=-100):
```

Comment:
```suggestion
def nll_loss(*, input, target, weight=None, reduce=None, reduction='mean', size_average=None, ignore_index=-100):
```

Just to keep the naming of these functions consistent